modules/serverless-create-inmemorychannel-yaml.adoc

```diff
@@ -3,22 +3,21 @@
 //  * serverless/event_workflows/serverless-channels.adoc
 
 [id="serverless-create-inmemorychannel-yaml_{context}"]
-= Creating a development channel using YAML
+= Creating a channel using YAML
 
 You can create a channel by using YAML with the cluster default configuration.
 
 .Prerequisites
 To create channels ensure that:
 
-* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
 
 .Procedure
 
-. Create a `Channel` object.
-.. Create a YAML file and copy the following sample code into it:
+* Create a `Channel` object.
+** Create a YAML file and copy the following sample code into it:
```

Comment:
I might rework this procedure so that it is an ordered set. Perhaps something like:

.Procedure

To create a `Channel` object:

. Create a YAML file
`...`
. Apply the YAML file:
`...`